src/wallet/scriptpubkeyman.cpp

```diff
@@ -1448,7 +1458,10 @@ void LegacyScriptPubKeyMan::MarkReserveKeysAsUsed(int64_t keypool_id)
         batch.ErasePool(index);
         WalletLogPrintf("keypool index %d removed\n", index);
         it = setKeyPool->erase(it);
+        result.push_back(keypool);
```

Comment:
I think this is better as `std::move(keypool)`?

Reply:
Done